packages/suite/src/views/wallet/coinmarket/exchange/offers/Offers/List/Quote/index.tsx

```diff
@@ -155,26 +160,26 @@ interface Props {
 }
 
 function getQuoteError(quote: ExchangeTrade) {
-    if (quote.min && Number(quote.sendStringAmount) < quote.min) {
+    const cryptoAmount = Number(quote.sendStringAmount);
+    const symbol = quote.send?.toLowerCase();
+    if (quote.min && cryptoAmount < quote.min) {
         return (
             <Translation
                 id="TR_OFFER_ERROR_MINIMUM_CRYPTO"
                 values={{
-                    amount: formatCryptoAmount(Number(quote.sendStringAmount)),
-                    min: formatCryptoAmount(quote.min),
-                    currency: quote.send,
+                    amount: <FormattedCryptoAmount value={cryptoAmount} symbol={symbol} />,
+                    min: <FormattedCryptoAmount value={quote.min} symbol={symbol} />,
                 }}
             />
         );
     }
-    if (quote.max && quote.max !== 'NONE' && Number(quote.sendStringAmount) > quote.max) {
+    if (quote.max && quote.max !== 'NONE' && cryptoAmount > quote.max) {
```

Comment:
`cryptoAmount` will have correct units here?

Reply:
No change here, it's value from Invity BE which is in BTC.